v3/plugins/redhat/quarkus/1.2.0/meta.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: v2
+publisher: redhat
+name: quarkus
+version: 1.2.0
+type: VS Code extension
+displayName: Quarkus Tools
+title: Quarkus Tools by Red Hat
+description: This plug-in provides support for Quarkus development via a Quarkus language server, Java jdt.ls and Java Debugger.
+icon: https://www.eclipse.org/che/images/logo-eclipseche.svg
+repository: https://github.com/redhat-developer/vscode-quarkus
+category: Language
+firstPublicationDate: "2019-11-25"
+spec:
+  containers:
+    - image: "docker.io/eclipse/che-remote-plugin-runner-java8:next"
+      name: vscode-quarkus
+      memoryLimit: "1500Mi"
+      volumes:
+      - mountPath: "/home/theia/.m2"
+        name: m2
+  extensions:
+    - https://github.com/microsoft/vscode-java-debug/releases/download/0.23.0/vscode-java-debug-0.23.0.vsix
```

Comment:
Should we also update https://github.com/eclipse/che-plugin-registry/blob/master/v3/plugins/redhat/java11/0.50.0/meta.yaml#L22 and https://github.com/eclipse/che-plugin-registry/blob/master/v3/plugins/redhat/java8/0.50.0/meta.yaml#L22 and https://github.com/eclipse/che-plugin-registry/blob/master/v3/plugins/redhat/java/0.50.0/meta.yaml#L22 to use this newer version of the debug plugin?

Reply:
I've created an issue for that. https://github.com/eclipse/che/issues/15560